pkg/controllers/provisioning/scheduling/node.go

```diff
@@ -102,3 +158,22 @@ func (n Node) hasCompatibleResources(resourceList v1.ResourceList, it cloudprovi
 	}
 	return true
 }
+
+func (n *Node) recalculateDaemonResources() {
+	n.daemonResources = nil
+	for _, ds := range n.daemons {
+		podRequirements := v1alpha5.NewPodRequirements(ds)
```

Comment:
consider inlining

Reply:
The function `recalculateDaemonResources`?  We call it from two places, once when we create the node and again after we add a pod.